pkgs/applications/editors/bluej/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, makeWrapper, jetbrains }:
```

Comment:
What if someone would like to make bluej use a different jdk? How complicated would it be to override it? How about using `jdk` as an argument here, and put in all-packages.nix this:

```nix
bluej = callPackage ../path {
  jdk = jetbrains.jdk;
};
```

Reply:
Note that upstream recommends using jdk11, where I don't know what jetbrains uses.

Reply:
Jetbrains is JDK11. As mentioned in the PR description, I tried with the normal jdk11 but I was unable to get past the splash screen (and the console output was entirely unhelpful, no errors or warnings at all).

Reply:
@charvp I'm not against using exactly `jetbrains.jdk`, I'm only suggesting to compose it a bit differently, so overrides will be easier, in case needed. I trust you that you've chosen wisely the right jdk flavor for this.

Reply:
See discussion about the subject at https://discourse.nixos.org/t/package-version-arguments-vs-generic-package-arguments/9071/3 .

Reply:
Done.